packages/react-jss/src/JssProvider.js

```diff
@@ -67,13 +67,18 @@ export default class JssProvider extends Component<Props> {
       sheetOptions.generateId = generateId
     } else if (!sheetOptions.generateId) {
       if (!this.generateId) {
-        let createGenerateId = createGenerateIdDefault
-        if (localJss && localJss.options.createGenerateId) {
-          createGenerateId = localJss.options.createGenerateId
+        if (localJss && localJss.generateClassName) {
+          // re-use the same classname generator pinned to the local jss instance
+          this.generateClassName = localJss.generateClassName
```

Comment:
This should be `this.generateId = localJss.generateId`

Reply:
localJss doesn't seem to have a generateId property at this stage.

![screenshot from 2018-11-20 09-40-13](https://user-images.githubusercontent.com/19770543/48739558-7700c480-eca8-11e8-804f-9e406cb8be2d.png)

Reply:
Yes, this is true for jss version below v10. We are currently in the process of polishing jss v10, and in this release, we renamed `generateClassName` to `generateId`

Reply:
If you run `yarn typecheck` you will some errors which correspond to that.